.github/workflows/workflow.yml

```diff
@@ -12,17 +12,17 @@ jobs:
       fail-fast: true
       matrix:
         os: ["ubuntu-latest"]  # On peut ajouter "macos-latest" si besoin
-        python-version: ["3.7.9", "3.8.9", "3.9.9"]
+        python-version: ["3.7.15", "3.8.14", "3.9.15"]
```

Comment:
Fixes CI as tested for example by https://github.com/openfisca/openfisca-france/pull/1930